### ISSUE TITLE
AI - Scientist for Non-Developers

### ISSUE THREAD

**Issue by Keegan-86** [SakanaAI/AI-Scientist#124]
Hey I'm really keen to explore the possibilities and potential implications of this application. But after going through the READ.ME I still have no idea how to set this up. Is there a way for you to explain it more simply for novice novice users.

**Comment by xprabhudayal** [comment]
what you wanted to know about? @Keegan-86

**Comment by Keegan-86** [comment]
Man I wanted to set up the Model and explore its possibilities. Ive tried
following the instructions but could never get passed pulling it through
Docker. Is there maybe a link that I could use?

**Comment by xprabhudayal** [comment]
try to use the conda env and a GPU instance with capabilities greater than **sm_80**, just install each and every dependency & it works fine on Linux..

**Comment by Tylersuard** [comment]
@Keegan-86 I am a professional developer with 6 years of experience, and even I can't get this repo to run.  The people who wrote this repo need to put it into an easy-to-use colab notebook, and have some better instructions for creating custom experiments.

**Comment by Tylersuard** [comment]
@conglu1997 we should not have to read an academic paper to figure out how to run your code on a custom experiment.  The instructions should be in the readme. You say it should be easy to understand, it is not, even for an experienced developer.

Please give us easy, detailed, step-by-step instructions on how to use your repo on other fields of scientific knowledge.  Your readme and the FAQs are not enough.

**Comment by xprabhudayal** [comment]
yeah its total chaos,  i just blindly run all the dependencies as im a begineer

**Comment by Keegan-86** [comment]
Thanks @Tylersuard I thought I was losing it. Look technically this is not my strength, but yeah its really confusing.

**Comment by erkinalp** [comment]
@Tylersuard it's actually a 186 page **report**, comprising a 30 page paper describing the AI Scientist itself and 11 papers generated by AI Scientist.

**Comment by Tylersuard** [comment]
> @Tylersuard it's actually a 186 page **report**, comprising a 30 page paper describing the AI Scientist itself and 11 papers generated by AI Scientist.

Thank you @erkinalp .  Anywhere in the paper, do they clearly explain how to modify the example.py file?

**Comment by BradKML** [comment]
I am kind of propelled to state the obvious @Tylersuard ... It is possible to get an AI development agent to get you documentation that reduces friction, so I would like to thank @erkinalp already making PRs for some of the things addressed. Just thinking how much more FOSS can things get.

**Comment by Krakaur** [comment]
@Tylersuard Hi, happy new year!

The AI Scientist is a non-profit initiative launched by a small group of exceptionally smart individuals. The general idea (as I understand it) is not to develop just a program but to create a community of scientists (or talented, capable individuals) who contribute to the automation of most scientific tasks. While a program may eventually emerge as a byproduct, the initiative is currently at the "proof of concept" stage.

In this regard, progress seems to be going sideways. Most participants have questions but are unable to provide answers. While a webinar might be helpful, in my opinion, a successful community requires organizers, clear task division, effective communication, and perhaps a Discord channel. Alternatively, implementing an AGILE framework with volunteers could work best.

I consider myself a hybrid, with roots in social sciences, natural sciences, and engineering. I am passionate about advancing AI into areas where it is most needed.
